.taskfiles/Talos/Taskfile.yaml

```diff
@@ -62,6 +60,38 @@ tasks:
     preconditions:
       - { msg: "Missing kubeconfig", sh: "test -f {{.KUBECONFIG_FILE}}" }
 
+  deploy:
```

Comment:
I would rename this to `bootstrap` and it also looks like the existing bootstrap task here isn't needed or left out?